### DISCUSSION TITLE
Table with sorting, selecting, etc.

### DISCUSSION THREAD

**Discussion by MichaelHipp** [saadeghi/daisyui#803] [Q&A] [answered]
Is there available anywhere a table component that has sorting by column, row selection, and other features that are fairly common in table components?

**Comment by saadeghi** [comment, marked as answer]
daisyUI is just the UI and it can't have sorting or other data manipulation functionalities.  
Sorting should be handled with JS.

**Reply by MichaelHipp** [reply]
Thanks. I guess the terminology is confusing. I'm most recently familiar with Vue + Vuetify where Vuetify has a table UI component with all that functionality bundled in.

**Reply by saadeghi** [reply]
It's more the language than terminology.  
daisyUI is just CSS (a plugin for Tailwind CSS). Can not do JS functionalities.
